CHANGELOG.md

```diff
@@ -4,20 +4,26 @@ This file is used to list changes made in each version of the haproxy cookbook.
 
 ## Unreleased
 
-## 12.1.0 - *2021-06-14*
+## 12.2.0 - _2021-06-21_
+
+- Add `use_promex` property to install.rb to support compiling with Prometheus Exporter support - [@Wicaeed](https://github.com/wicaeed)
+
+#
+
+## 12.1.0 - _2021-06-14_
 
 - Add `ssl_lib` and `ssl_inc` properties to `haproxy_install` to support openssl - [@derekgroh](https://github.com/derekgroh)
 
-## 12.0.1 - *2021-06-01*
+## 12.0.1 - _2021-06-01_
```

Comment:
Guessing you've auto fixed these with whatever MDL rules you have, they need to be reverted